swarm-production/docker-stack.yml

```diff
@@ -94,6 +93,79 @@ services:
         delay: "10s"
         monitor: "5s"
 
+  livy:
+    image: scality/clueso-livy:pensieve
+    hostname: livy
+    ports:
+      - "8998:8998"
+      - "4040-4056:4040-4056"
+      - "5005:5005"
+    expose:
+      - "8998"
+      - "10800"
+      - "11211"
+      - "47400-47500"
+      - "47100"
+    depends_on:
+      - spark-master
+    networks:
+      - backend
+      - frontend-dmz
+
+  spark-master:
+    image: scality/clueso-spark-master:pensieve
+    hostname: spark-master
+    depends_on:
+      - queue
+      - graphite
+      - lb
+    ports:
+      - "8080:8080"
+      - "7077:7077"
+    volumes:
+      - "heapdumps:/clueso/heapdumps:rw"
```

Comment:
@RachedBenMustapha, is keeping this in offensive?

Reply:
that's fine :)